cli/args/mod.rs

```diff
@@ -810,11 +812,14 @@ impl CliOptions {
 
     let maybe_lockfile = maybe_lockfile.filter(|_| !force_global_cache);
     let root_folder = start_dir.workspace.root_folder_configs();
+    let deno_dir_provider =
+      Arc::new(DenoDirProvider::new(flags.cache_path.clone()));
```

Comment:
`deno_dir_provider` is not `Deferred` anymore. The `Deferred` defined in `cli/factory.rs` uses `once_cell::unsync::OnceCell` and not thread-safe, that is inconvenient for `CliOptions`. Should I create `once_cell::sync::OnceCell`-version of `Deferred` and wrap this `deno_dir_provider`? (That feels a bit overkill to me for now)

Reply:
Nope. This is good. Thanks!